src/theory/bv/bitblast/simple_bitblaster.cpp

```diff
@@ -53,6 +53,10 @@ void BBSimple::storeBBTerm(TNode node, const Bits& bits)
 
 bool BBSimple::hasBBAtom(TNode atom) const
 {
+  if (atom.getKind() == kind::NOT)
```

Comment:
Can you rename the argument to this method `lit`?  When I see `atom` as an argument this usually needs non-negated.